### DISCUSSION TITLE
[ニュートラル/Neutral]悪徳探偵？(ニュートラル版ゲッサー)

### DISCUSSION THREAD

**Discussion by yukieiji** [yukieiji/ExtremeRoles#296] [Ideas]
- アイディアが完全に固まりきってないので意見求めも含めてディスカッション化
  - これらの能力、仕様は変更されることがあります

## 能力詳細
- 名前：悪徳探偵？
- 色：不明
- 陣営：ニュートラル

- 役職コンセプト：ニュートラル版ゲッサー

- 勝利条件：
  1. 決められた回数「能力推察」を成功させる
  2. 自分自身を除いた生きているプレイヤー全員の「能力推察」を成功させる

- タスクなし、マップ機器は全て使用不可、サボタージュ修理不可、ボタンは押せる

- 会議能力：「能力推察」
  - 会議中のプレイヤー1人を対象に発動
  - その人の役職を選択
    - 「能力推察」を行えるのはプレイヤー1人に対してゲーム中1回のみ、同じ役職の推察は1会議中一回のみ
      - これによって役職の総当たりによる勝利を防ぐ
   - 成功：全員の会議チャット + SE
   - 失敗：その人への「身辺調査」への発動ペナルティ + 「身辺調査」のクールタイムペナルティ
- ボタン能力「身辺調査」
  - プレイヤー一人を対象にとって指定秒数近くにいることで発動
  - その人の役職に関する断片的な情報を入手できる
    - 1回目：？？？
    - 2回目：陣営
    - 3回目：役職

- コンビネーション役職の「探偵」と同じ能力を持つ(犯行現場等)

- 他の陣営への影響
  - CO抑制が強力にかかかる

## 未検討事項
- この役職の勝利を止める方法が無さすぎる、クルーはボディーガードで防げるようにするのは十分にありだがインポスターや第三陣営がこの役職に対抗する手段が存在し得ない
  - インポスターはキルして落とす先方が使えるかもしれないがしつこく付き纏われるとかなりキツイかもしれない

**Comment by flog4624** [comment]
悪徳探偵（仮名）の勝利を止める方法として「役職当てを外したペナルティとして、一定確率で全員に自分が悪徳探偵だということと何回当てたかがバレる」はどうでしょうか。
そして外した回数が多いほどバレる確率が上がると、緊張感が出て良いと思います。
例
そのゲーム中初めて外した時は30%でバレる
2回目は65%
3回目は100%。つまり確実にバレる≒勝ち目がなくなる

**Reply by yukieiji** [reply]
かなり返信が遅れましたが、確率で外れるとなると戦術の幅が狭くなる(ガチガチに推理してから役職を当てる必要 + 絶対にないという役職をわざと使用する戦術)ので個人的にはやりたくないって感じです

**Reply by flog4624** [reply]
それでしたらこちらはどうでしょうか。探偵というからにはバレずに尾行しなければならないというイメージから、悪徳探偵の勝利を食い止める方法として提案しました
・悪徳探偵が勝利条件を満たした会議の後に、アサシン会議のように特殊会議が発動
・テキストチャットはその時の生存者全員と「悪徳探偵によって最後に役職を当てられたプレイヤー」が参加できるが、投票権を持つのは「悪徳探偵によって最後に役職を当てられたプレイヤー」ただ1人。最後に役職を当てられたプレイヤーだけは死亡していても参加でき、投票できる
・その投票で悪徳探偵を当てられたら悪徳探偵の勝利は無効となり、悪徳探偵は死亡する
・投票で悪徳探偵を当てられなかったらそのまま悪徳探偵が勝利

**Reply by yukieiji** [reply]
「勝利を止める」という部分は達成できてるとは思いますが結局、この役職自体を推理することが難しいという部分が変わってないので一時的な時間稼ぎ(特殊会議を発生させたとしても止められる可能性は低い気がします)にしかなっていない気がしています
「勝利が止められない」というのは役職の推理ができないという部分から来ているのでそこを緩和しなければならないと思います
